config/README.md

```diff
@@ -1,176 +1,344 @@
 
-### Preparation
+## Configuration files
 
-The script needs a `venv` with some packages installed:
+In a nutshell what `plone/meta` does
+is to put some configuration files on the repository.
 
-```shell
-python3.11 -m venv .
-bin/pip install -r requirements.txt
+Currently, the files being managed by `plone/meta` are:
+
+- `.meta.toml`: `plone/meta`'s configuration file
+- `.editorconfig`: configuration meant to be read by IDEs
+- `.flake8`: [`flake8`](https://pypi.org/project/flake8) configuration
+- `.gitignore`: list of file/folders patterns that `git` should ignore
+- `.github/workflows/meta.yml`: GitHub Actions to run the testing and QA tools on GitHub
+- `.pre-commit-config.yaml`: [`pre-commit`](https://pypi.org/project/pre-commit) configuration
+- `pyproject.toml`: configuration options for a wide variety of python tooling
+- `tox.ini`: [`tox`](https://pypi.org/project/tox) configuration, __the most important file__
+
+You can find the _template_ files for each of these files
+in the `config/default` folder of this `plone/meta` repository.
+
+You will notice that they have a `.jinja` extension,
+that's because the files are not merely copied over to the target repository,
+but rather they are enhanced/adapted on the way there.
+
+See the next section about how to extend/modify these configuration files.
+
+## Configuration options
+
+One thing is trying to standardize,
+but another thing is being flexible enough
+to adapt to each repositories' special needs.
+
+Fortunately `plone/meta` tries its best to accomplish both:
+
+- it provides sane defaults
+- it allows to extend the defaults with custom configuration
+
+The files mentioned above have comments all over them
+with instructions on how to extend/modify/influence
+what `plone/meta` ends up adding on those files.
+
+Those options are to be stored,
+as it is mentioned on the comments themselves,
+on `.meta.toml`.
+
+This way, when the configuration files get re-generated,
+`plone/meta` reads the configuration on `.meta.toml`
+and re-applies the specific configuration on the other files.
+
+See the specific configuration files sections below on how to extend/modify each configuration file.
+
+### Configuration philosophy
+
+The idea behind the configuration system
+in `plone/meta` controlled configuration files is to make it as simple as possible.
+
+Rather than adding plenty of configuration options,
+almost all configuration files have a `extra_lines`
+that allows you to paste as much configuration as you want there.
+
+This way providing a simple, yet powerful, extension mechanism.
+
+There are a few, and growing, other configuration options on for a few files,
+where the simple approach described above is not enough.
+
+### More configuration options
+
+Please, use the [issue tracker](https://github.com/plone/meta/issues/new)
+to ask for more extension points whenever `plone/meta`
+does not fulfill all your needs.
+
+### `.editorconfig`
+
+```
```

Comment:
All TOML examples should use the `toml` lexer.

````suggestion
```toml
````

Reply:
Thanks, done!